lms/views/lti/basic_launch.py

```diff
@@ -202,6 +206,27 @@ def _record_course(self):
             user=self.request.user, groups=[self.context.course]
         )
 
+    def _configure_js_for_file_picker(self):
+        """
+        Show the file-picker for the user to choose a document.
+
+        This happens if we cannot resolve a document URL for any reason.
+        """
+
+        if not self.request.has_permission(Permissions.LTI_CONFIGURE_ASSIGNMENT):
+            # Looks like the user is not an instructor, so show an error page
+
+            # https://docs.pylonsproject.org/projects/pyramid/en/latest/narr
+            # /renderers.html?highlight=override_renderer#overriding-a-renderer-at-runtime
+            self.request.override_renderer = "lms:templates/lti/basic_launch/unconfigured_launch_not_authorized.html.jinja2"
+            return {}
```

Comment:
Hmm, yeah. Is `unconfigured_launch_not_authorized.html.jinja2` gonna work at all here? Isn't the request a JavaScript API request from the frontend rather than a normal browser page load request?

Reply:
Yeah this is fugly, tracking it here as it needs coordination in with the FE: https://github.com/hypothesis/lms/issues/5159

In practice is not that important as this is called when we determined before that's ok to do so. It needs fixing but just thinking about the testing instructions (probably hard coding something in that permissions check) suggest it's not going to be a common occurrence.